src/server/system_services/bucket_server.js

```diff
@@ -715,53 +705,52 @@ async function update_buckets(req) {
  */
 function update_bucket_s3_access(req) {
```

Comment:
I would remove that. the user only option to update the bucket s3 access will be using the bucket policy api. I think this was used only by the UI anyhow.

Reply:
Got it, will remove this ✅ .